doc/source/user_guide/index.rst

```diff
@@ -208,6 +208,18 @@ You can control the global logging level at any time with:
 
 .. code:: python
 
-    import ansys.fluent.core as pyfluent
-    pyfluent.set_log_level("DEBUG") # by default, only errors are shown
+    from ansys.fluent.core import logging
```

Comment:
@acarvalh-work this is just a standard Python module that can be imported directly. It happens to be available via ansys.fluent.core. @raph-luc is updating logging generally and I suspect that he will be completely removing this stuff.